docs/00-welcome.mdx

```diff
@@ -5,7 +5,7 @@ path: /docs
 
 ### What is Excalibur?
 
-Excalibur is a **free [open source](https://github.com/excaliburjs/Excalibur) 2D game engine** written in **[TypeScript](https://www.typescriptlang.org/)** for the web written in TypeScript for making 2D games in HTML5 canvas.
+Excalibur is a **free [open source](https://github.com/excaliburjs/Excalibur) 2D game engine** written in [TypeScript](https://www.typescriptlang.org/).
 
 Our goal is to make it easier for you to create 2D HTML/JS games, whether you're new to game development or you're an experienced game developer. We take care of all of the boilerplate engine code, cross-platform targeting, and more! Use as much or as little as you need!
 
```

Comment:
Personally, I find the gifs on the main page a little distracting when I'm trying to read the text. Could we consider replacing them with static images, or disabling auto-play by default?

Reply:
@jedeen Sure, I'm down. Perhaps a static image of the logo and play button?